### PR TITLE
Adds wrapped BigNumber constructor

### PR DESCRIPTION
**Pull request by bthaile**
Here the number comes back greater than 15 digits in length. AN will send this as a string in the future, but causes UI to blow up. so adding it.

Putting changes needed for BigNumber here.

## Submitter checklist
- [ ] Test covering functionality added/fixed in
- Check the linked story includes the following:
  - [ ] Steps to reproduce
  - [ ] Screenshot (If applicable)

## Reviewer Checklist
- [ ] Test/lint pass 
- [ ] Post merge, story marked complete

### CODE REVIEW

**Comment by tinybike** [comment]
Updated PR title to better reflect contents.

**Comment by tinybike** [comment]
Has FullWrappedBigNumber been tested to make sure that it doesn't subtly break BigNumbers in some way?  i.e., that `BigNumber.isBigNumber(aFullWrappedBigNumber) === true`, `FullWrappedBigNumber.constructor === BigNumber`, etc.?

**Comment by pgebheim** [comment]
Agreed with @tinybike  -- I think the main issue with actually wrapping the BN constructor is that we have logic which checks that the instance.constructor === a certain function in some places in the code.

This would need to be replaced by BigNumber.isBigNumber everywhere for this wrapper to work.

So:
[] - Add a test that .isBigNumber works

and I would recommend instead of using WrappedBigNumber as a class, just have a function which does what this constructor does and return a real BigNumber instance so constructor checks will work.

**Comment by justinbarry** [comment]
Updated. Please review.

**Comment by justinbarry** [comment]
@pgebheim @tinybike @bthaile Bump.

File: src/utils/wrapped-big-number.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    newBigNumber = new BigNumber(...args)
+  } catch (e) {
+    logError('Error instantiating WrappedBigNumber', e)
+  }
+
+  return newBigNumber
+}
+
+// Note this is exported from here.
+export { default as BigNumber } from 'bignumber.js'
```
Comment (tinybike) [inline src/utils/wrapped-big-number.js]: Why is this re-exported?  If using BigNumber directly, it seems simpler to just `import BigNumber from 'bignumber.js'`.

File: src/modules/create-market/components/create-market-review.jsx [hunk 2]
```diff
@@ -3,7 +3,7 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import { augur } from 'services/augurjs'
-import BigNumber from 'bignumber.js'
+import BigNumberLib from 'bignumber.js'
```
Comment (tinybike) [inline src/modules/create-market/components/create-market-review.jsx]: Suggest using `import BigNumber from 'bignumber.js'` to keep the naming consistent.

**Comment by justinbarry** [comment]
Broke this into multiple PRs. Closing this one.

**Pull request closed without merging**